app/Http/Controllers/DashboardController.php

```diff
@@ -40,6 +41,34 @@ public function getExpenses()
     public function getExpenseSummary($expenses){
       //make an summary
       //return new
+        $summary = array();
+        $owed = DB::table('expenses AS e')
+            ->leftjoin('sharedexpenses AS se', 'e.id', '=', 'se.expense_id')
```

Comment:
I think the left join is unnecessary here and possibly dangerous as it could introduce null values. (if you just joined, then you get only the expenses where a settled expense exists (inner join) it would remove the personal expenses naturally.